test/functional/rpc_dumptxoutset.py

```diff
@@ -19,6 +19,17 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
 
+    def check_expected_network(self, node, active):
+        rev_file = node.blocks_path / "rev00000.dat"
+        bogus_file = node.blocks_path / "bogus.dat"
+        rev_file.rename(bogus_file)
+        assert_raises_rpc_error(
```

Comment:
nit:
```suggestion
        self.log.info(f"Test that dumptxoutset with rollback type fails given an invalid height")
        assert_raises_rpc_error(
```

Reply:
Hm, I don't think it makes sense to add that. First, the actual error here is not the point of this test. This test is about the network activity and even if it covers something else I would like that to be structured differently then and have the logs be in `run_test`. Where it is now it would also be printed twice. Second, the content isn't right: The reason this fails is not the rollback height, it's because the rev file can not be read, so the rolling back action failed as in, it could not reach the requested height. But if the re file wasn't moved above there should be no error raised.

Reply:
Yes, that's correct. Thanks! I got confused by the error message, block height was actually valid (`COINBASE_MATURITY = 100`), then we are missing testing the rollback type given an invalid height (?).

Reply:
Seems like we don't have a test for that explicitly. It's handled by `ParseHashOrHeight()` which used in other RPCs and is covered in other tests but still would be good to have it for `dumptxoutset`. Do you want to open a PR for that @pablomartin4btc ? Otherwise, I can add it here.

Reply:
I can open a new PR to test the rollback type option with both a valid and invalid heights. I'll do as soon as this one gets merged. Thanks.